### PR TITLE
actions: Add some additional labels for devicetree

### PR DESCRIPTION
**Pull request by galak**
Add labeling of PRs for 'area: Device Tree Binding' (files under
dts/bindings/*)  and 'area: Device Tree Tooling' (files under
scripts/dts/*).

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

### CODE REVIEW

**Review by nashif — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -21,6 +21,11 @@
   - "**/*.dts"
   - "include/devicetree/*"
   - "include/dt-bindings/**/*"
+  - "scripts/dts/**/*"
```
Comment (erwango) [inline .github/labeler.yml:24]: Isn't it duplicated with "Device Tree Tooling" below?

Comment (galak) [reply .github/labeler.yml:24]: Want to have the 'Area: Device Tree' label as a catch all.

Comment (pabigot) [reply .github/labeler.yml:24]: Should we have `Area: Device Tree API` for `include/*`?

**Review by erwango — APPROVED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (mbolivar-nordic) [reply .github/labeler.yml:24]: >  Should we have Area: Device Tree API for include/*?

I think that should be `include/devicetree.h`, right?

File: .github/labeler.yml [hunk 2]
```diff
@@ -21,6 +21,11 @@
...
+"area: Device Tree Binding":
+  - "dts/bindings/**/*"
+"area: Device Tree Tooling":
```
Comment (mbolivar-nordic) [inline .github/labeler.yml:27]: While you're here, is there any chance of renaming these "Devicetree" instead? It's OK if that's out of scope for this PR.

File: .github/labeler.yml (hunk 1 above)
Comment (pabigot) [reply .github/labeler.yml:24]: I meant `Area: Devicetree API` for the files under `include/` that are relevant to devicetree:

    - "include/devicetree/*"
    - "include/devicetree.h"

not everything under `include/`.  (I agree about the spelling w/o the space.)

File: .github/labeler.yml (hunk 2 above)
Comment (galak) [reply .github/labeler.yml]: Added include/devicetree.h and renamed the labels.

**Comment by galak** [comment]
We can add a 'area: Devictree API' as a followup PR.

**Review by mbolivar-nordic — APPROVED** [review]

**Comment by galak** [comment]
Based on @nashif comment, tweaked this so we should avoid duplication of 'area: Devicetree' going forward.

**Review by mbolivar-nordic — APPROVED** [review]

**Pull request merged**